spec/migrations/20170131160216_migrate_ansible_tower_configuration_manager_sti_type_to_automation_manager_spec.rb

```diff
@@ -0,0 +1,31 @@
+require_migration
+
+class ExtManagementSystem < ActiveRecord::Base
+  self.inheritance_column = :_type_disabled
+end
```

Comment:
So once I changed to the `configuration_manager.reload.type`, the above error won't happen even without the definition of `ExtManagementSystem` here now. 
Removed now. Thanks